templates/ceph/what-is-ceph.html

```diff
@@ -33,307 +33,341 @@ <h1>
+      <div class="col-7">
+        <p>
+          Ceph makes it possible to decouple data from physical storage hardware using software abstraction layers, which provides unparalleled scaling and fault management capabilities. This makes Ceph ideal for cloud, <a href="/openstack">Openstack</a>, <a href="/kubernetes">Kubernetes</a>, and other microservice and container-based workloads, as it can effectively address large data volume storage needs.
+        </p>
+        <p>
+          The main advantage of Ceph is that it provides interfaces for multiple storage types within a single cluster, eliminating the need for multiple storage solutions or any specialised hardware, thus reducing management overheads.
+        </p>
+        <p>
+          Use cases for Ceph range from private cloud infrastructure (both hyper-converged and disaggregated) to big data analytics and rich media, or as an alternative to public cloud storage.
+        </p>
       </div>
-    </section>
+    </div>
+  </div>
+</section>
 
-    <section class="p-strip">
-      <div class="row">
-        <div class="col-8">
-          <h2>What is a Ceph cluster?</h2>
-          <p>
-            A Ceph storage cluster consists of the following types of daemons:
-          </p>
-          <ul class="p-list">
-            <li class="p-list__item">
-              Cluster monitors (<code>ceph-mon</code>) that maintain the map of the cluster state, keeping track of active and failed cluster nodes, cluster configuration, and information about data placement and manage daemon-client authentications.</li>
-              <li class="p-list__item">
-                Object storage devices (<code>ceph-osd</code>) that store data on behalf of Ceph clients. Additionally, they handle data replication, erasure coding, recovery, rebalancing, monitoring and reporting.
-              </li>
-              <li class="p-list__item">
-                Managers (<code>ceph-mgr</code>) that maintain cluster runtime metrics, enable dashboarding capabilities and provide an interface to external monitoring systems.
-              </li>
-              <li class="p-list__item">
-                Metadata servers (<code>ceph-mds</code>) that store metadata on behalf of the Ceph File System, mapping filenames and directories of the file system to RADOS objects and allow the use of POSIX semantics to access the files.
-              </li>
-            </ul>
-            <p>
-              Ceph stores data as objects within logical storage pools. In order to achieve scalability, rebalancing and recovery capabilities, Ceph shards the pools into placement groups. The CRUSH algorithm then defines the placement group for storing an object and thereafter calculates which Ceph OSD Daemon should store the placement group.
-            </p>
-          </div>
-        </div>
-      </section>
+<section class="p-strip">
+  <div class="row">
+    <div class="row u-equal-height">
+      <h2>What is a Ceph cluster?</h2>
```

Comment:
Would it be better if there is some space between the title and the image?

Reply:
Added